tests/tasks/aws/test_step_function.py

```diff
@@ -4,31 +4,27 @@
 
 pytest.importorskip("boto3")
 
-import prefect
 from prefect.tasks.aws import StepActivate
-from prefect.utilities.configuration import set_temporary_config
 
 
 class TestStepActivate:
     def test_initialization(self):
-        task = StepActivate(state_machine_arn="arn", execution_name="name")
+        task = StepActivate(state_machine_arn="arn")
 
     def test_initialization_passes_to_task_constructor(self):
         task = StepActivate(
-            state_machine_arn="arn", execution_name="name", name="test", tags=["AWS"]
+            state_machine_arn="arn", name="test", tags=["AWS"]
         )
         assert task.name == "test"
         assert task.tags == {"AWS"}
 
     def test_exposes_boto3_start_execution_api(self, monkeypatch):
-        task = StepActivate(state_machine_arn="arn", execution_name="name")
+        task = StepActivate(state_machine_arn="arn")
         client = MagicMock()
         boto3 = MagicMock(client=client)
         monkeypatch.setattr("prefect.utilities.aws.boto3", boto3)
-        task.run()
+        task.run(execution_name="name")
 
         called_method = client.mock_calls[1]
         assert called_method[0] == "().start_execution"
-        called_method.assert_called_once_with(
-            {"stateMachineArn": "arn", "name": "name", "input": {}}
-        )
+        assert called_method[2] == {"stateMachineArn": "arn", "name": "name", "input": "{}"}
```

Comment:
`called_method.assert_called_once_with` pass for me with any input. :grimacing: For example, this would pass the test:
```python
called_method.assert_called_once_with(
    {"foo": "bar"}
)
```
I replaced it with comparing the kwargs directly. It no longer assert anymore that there was only a single `start_execution` call, and I'm also not a fan of the magic numbers 1, 0, and 2 that are used, but I couldn't find a better solution right away.

Reply:
Also note that the old test was incorrectly asserting that `"input"` should be `{}`. It actually is the string `"{}"`.